### DISCUSSION TITLE
✅ 🚀 Feature Request: Disable Collect Submissions by Default

### DISCUSSION THREAD

**Discussion by mcrowell** [elementor/elementor#21820] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Since the upgrade to Elementor Pro 3.12.0 Collect Submissions is on by default for new and existing forms.
This can bloat database sizes if a form gets hit with a lot of spam.

### Describe the solution you'd like

Don't enable Collect Submissions by default on new and existing forms.

### Describe alternatives you've considered

Give us the option to enable Collect Submissions when we want.

**Comment by hitsgee** [comment]
Or the option to auto delete after a set period? Like 30 days.

**Comment by black-eye** [comment]
+1 I don’t use Elementor forms at all. No reason to have this enabled. At all. Especially without a way to disable it.

**Comment by nicholaszein** [comment]
Hello there,

This issue was resolved in Elementor Pro **v3.12.1**! 🥳
